nemo/collections/asr/jasper.py

```diff
@@ -114,6 +117,23 @@ def output_ports(self):
             "encoded_lengths": NeuralType(tuple('B'), LengthsType()),
         }
 
+    @property
+    def disabled_deployment_input_ports(self):
+        return set(["length"])
+
+    @property
+    def disabled_deployment_output_ports(self):
+        return set(["encoded_lengths"])
+
+    def prepare_for_deployment(self):
+        print("prepare for deployment called")
```

Comment:
please do not use "print" - use logger instead

Reply:
Yep that was vestigial, sorry. Removing. We don't need to log anything at all at that spot.